src/app/cube/shared/learning-object/learning-object.component.ts

```diff
@@ -142,7 +142,8 @@ export class LearningObjectListingComponent implements OnInit, OnChanges, OnDest
       this.learningObject.collection !== 'intro_to_cyber' &&
       this.learningObject.collection !== 'secure_coding_community' &&
       this.learningObject.collection !== 'plan c' &&
-      this.learningObject.collection !== 'max_power'
+      this.learningObject.collection !== 'max_power' &&
+      this.learningObject.collection !== 'Drafts'
```

Comment:
```suggestion
      this.learningObject.collection !== ''
```
Learning objects that do not belong to a collection just have an empty string so just checking for the empty string should fix the logo issue